tests/models/test_baseoperatormeta.py

```diff
@@ -0,0 +1,64 @@
+import os
+from typing import Any
+from unittest import TestCase
+from unittest.mock import Mock, patch, MagicMock
+
+import pendulum
+from sqlalchemy.orm import Session
+
+from airflow import AirflowException, DAG
+from airflow.models import TaskInstance, DagRun
+from airflow.models.baseoperator import BaseOperator
+from airflow.utils.context import Context
+from airflow.utils.state import DagRunState
+
+
+class HelloWorldOperator(BaseOperator):
+    called = False
+
+    def execute(self, context: Context) -> Any:
+        HelloWorldOperator.called = True
+        return f"Hello {self.owner}!"
+
+
+class IterableSession(Session):
+    def __next__(self):
+        pass
+
+
+class BaseOperatorMetaTestCase(TestCase):
+
+    @patch.dict(os.environ, {'AIRFLOW__CORE__UNIT_TEST_MODE': 'False'})
```

Comment:
We have mocker config helper in `tests.test_utils.config.conf_vars`

Some samples

https://github.com/apache/airflow/blob/8f1418f4de89b334f188d7d5a3b0f348ad4a0372/tests/cli/commands/test_config_command.py#L167-L173

https://github.com/apache/airflow/blob/1aa91a482630f389005a9a36b0f038e675163411/tests/www/views/test_views_robots.py#L31-L40